modules/policy-criteria.adoc

```diff
@@ -29,524 +30,831 @@ In this table:
-However, on earlier versions you can still use regular expressions for the fields listed in the *Regular expressions* column.
-====
-
-[cols="<,<,<,^,^,^,<"]
+[cols="<,<,<,<,^,<"]
 |===
-| *Attribute* | *Description* | *RHACS version* | *Regular expressions* | *NOT* | *AND, OR* | *Phase*
+| *Attribute* | *Description* | *JSON Attribute* | *Allowed Values* | *Regex*, *NOT*, *AND, OR* | *Phase*
 
-| Namespace
-| The name of the namespace.
-| 3.0.51 and newer
-| ✓
-| ✓
-| ✓
-| Deploy
+6+| *Section: Image registry*
 
 | Image Registry
 | The name of the image registry.
-| All
-| ✓
-| ✓
-| ✓
-| Deploy
-
-| Image Remote
+| Image Registry
+| String
+| Regex, +
+NOT, +
+AND, OR
+| *Build*, +
+*Deploy*, +
+*Runtime* (when used with a Runtime criterion)
+
+| Image Name
 | The full name of the image in registry, for example `library/nginx`.
-| All
-| ✓
-| ✓
-| ✓
-| Deploy
+| Image Remote
+| String
+| Regex, +
+NOT, +
+AND, OR
+| *Build*, +
+*Deploy*, +
+*Runtime* (when used with a Runtime criterion)
 
 | Image Tag
 | Identifier for an image.
-| All
-| ✓
-| ✓
-| ✓
-| Deploy
-
-| Days since image was created
-| The number of days from image creation date.
-| All
-| ✕
-| ✕
-| ✕
-| Build
-
-| Days since image was last scanned
-| The number of days since the last image scan.
-| All
-| ✕
-| ✕
-| ✕
-| Build
+| Image Tag
+| String
+| Regex, +
+NOT, +
+AND, OR
+| *Build*, +
+*Deploy*, +
+*Runtime* (when used with a Runtime criterion)
+
+| Image Signature
+| The list of signature integrations you can use to verify an image's signature. Create alerts on images that either do not have a signature or their signature is not verifiable by at least one of the provided signature integrations.
+| Image Signature Verified By
+| A valid ID of an already configured image signature integration
+| ! `OR` only
+| *Build*, +
+*Deploy*, +
+*Runtime* (when used with a Runtime criterion)
+
+
+6+| *Section: Image contents*
+
+| Image age
+| The minimum number of days from image creation date.
+| Image Age
+| Integer
+| ✕
+| *Build*, +
+*Deploy*, +
+*Runtime* (when used with a Runtime criterion)
+
+| Image scan age
+| The minimum number of days since the image was last scanned.
+| Image Scan Age
+| Integer
+| ✕
+| *Build*, +
+*Deploy*, +
+*Runtime* (when used with a Runtime criterion)
+
+| Image User
+| Matches the USER directive in the Dockerfile. See https://docs.docker.com/engine/reference/builder/#user for details
+.
+| Image User
+| String
+| Regex, +
+NOT, +
+AND, OR
+| *Build*, +
+*Deploy*, +
+*Runtime* (when used with a Runtime criterion)
 
 | Dockerfile Line
 | A specific line in the Dockerfile, including both instructions and arguments.
-| All
-| ! only for values
-| ✕
-| ✓
-| Build
-
-| Image is NOT Scanned
-| No scan data is available for the image.
-| All
-| ✕
-| ✕
-| ✕
-| Build
+| Dockerfile Line
+| One of: LABEL, RUN, CMD, EXPOSE, ENV, ADD, COPY, ENTRYPOINT, VOLUME, USER, WORKDIR, ONBUILD
+| ! Regex only for values, +
```

Comment:
just making sure the symbols are OK, like `!` and `+` should be displaying

Reply:
! is displayed, and is supposed to be.

The + sign at the end of lines are the adoc way of forcing a newline within a single table cell, and they are not displayed.